zhaquirks/tuya/tuya_siren.py

```diff
@@ -98,3 +202,254 @@ class TuyaSirenRingtone(t.enum8):
     .skip_configuration()
     .add_to_registry()
 )
+
+
+# Neo NAS-AB02B0 - Temperature & humidity sensor and alarm
+(
+    TuyaQuirkBuilder("_TZE200_d0yu2xgi", "TS0601")
+    .applies_to("_TYST11_d0yu2xgi", "0yu2xgi")
+    .tuya_enum(
+        dp_id=101,
+        attribute_name="power_type",
+        enum_class=NeoBatteryState,
+        translation_key="power_type",
+        fallback_name="Power type",
+    )
```

Comment:
Should this be a sensor?

Reply:
Yes, it should be.

Reply:
https://github.com/zigpy/zha-device-handlers/pull/3741/commits/f5b13d8193324cd107f22188ad5a127f86f9a74e should address this.